index.html

```diff
@@ -130,6 +137,44 @@ <h1 class="text-center">About Us</h1>
     </section>
     <!-- about ends -->
 
+    
+<!--issue.104 starts-->
+<section id="issue.104" class="issue.104">
+<div class="container">
+    <h1 class="text-center">Sharpen Your Skills</h1>
+    <hr>
+<br><br>
+<table class="center">
+  <tr>
+    <th><i class="fas fa-atom"></i>&nbsp; <a href="  ">Data Structures</a></th>
```

Comment:
If you are not providing any link give # instead of empty.

Reply:
done